WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -315,13 +315,23 @@ private async Task<(ImmutableArray<AliceClient> aliceClientsThatSigned, IEnumera
 		try
 		{
 			var roundId = roundState.Id;
+			var roundParameters = roundState.CoinjoinState.Parameters;
 
 			registeredAliceClientAndCircuits = await ProceedWithInputRegAndConfirmAsync(smartCoins, roundState, cancellationToken).ConfigureAwait(false);
 			if (!registeredAliceClientAndCircuits.Any())
 			{
 				throw new CoinJoinClientException(CoinjoinError.NoCoinsEligibleToMix, $"The coordinator rejected all {smartCoins.Count()} inputs.");
 			}
 
+			var smallestEffectiveDenom = roundParameters.CalculateSmallestReasonableEffectiveDenomination();
+			var effectiveInputSum = registeredAliceClientAndCircuits
+				.Select(x => x.AliceClient.SmartCoin)
+				.Sum(x => x.EffectiveValue(roundParameters.MiningFeeRate));
```

Comment:
I think you should subtract the coordination fee, but it looks like you didn't.

By the way, you can use `AliceClient.EffectiveValue`.

Reply:
Yes, thanks for noticing. Fixed!